hardware_enablement/kmm-release-notes.adoc

```diff
@@ -28,4 +27,297 @@ toc::[]
+
+** *Fix*: Some adjustments were made in Konflux.
+
+** *Result:* The KMM Operator build now includes the Operator version and git commit in the log files.
+
+// MGMT-20754
+* The KMM Operator does not load the module after node with taint is rebooted.
+
+** *Cause*: The Kernel Module Management (KMM) Operator does not reload the kernel module in case the node reboot sequence is too quick. The reboot is determined based on the timestamp of the status condition being later than the timestamp in the Node Machine Configuration (NMC) status.
+
+** *Consequence*: When the reboot happens quickly, in less time than the grace period, the node state will not change. After the node reboots, KMM will not load the kernel module again.
+
+** *Fix*: Instead of relying on the condition state, NMC can rely on the `Status.NodeInfo.BootID` field. This field is set by kubelet based on the /proc/sys/kernel/random/boot_id file of the server node, so it is updated after each reboot.
+
+** *Result:* The more accurate timestamps enable the Kernel Module Management (KMM) Operator to reload the kernel module after the node reboot sequence.
+
+// MGMT-20775
+* Redeploying a module that uses in-cluster builds fails with the `ImagePullBackOff` policy.
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, the `pullPolicy` for the puller pod and the worker pod is different.
+
+** *Consequence*: An image can be considered as existing while it fact it is not.
+
+** *Fix*: Make the pull policy of the pull pod the same as the pull policy defined in the KMM Module, as this is what is used by the worker pod.
+
+** *Result:* The MIC will represent the state of the image in the same way the worker pod will be able to access it.
+
+// MGMT-20785
+* The MIC controller creates two pull-pods when it should create just one.
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, the `ModuleImagesConfig` (MIC) controller may create multiple pull-pods for the same image.
+
+** *Consequence*: Resources are wasted and confusion is caused.
+
+** *Fix*: The `CreateOrPatch` MIC API receives a slice of `ImageSpecs`, as the input is created by going over the the target nodes and adding their images to the slice, so any duplicate `ImageSpecs`, are now filtered out.
+
+** *Result:* The KMM Operator works as expected.
+
+// MGMT-20827
+* The `job.dcDelay` example in the documentation should specify `0s` instead of 0.
+
+** *Cause*: The Kernel Module Management (KMM) Operator default `job.gcDelay` duration field is 0s but the documentation says it is `0`.
+
+** *Consequence*: This can lead a customer in the wrong direction setting a custom value of 60 instead of 60s or 1m that will resolve in an error due to the wrong input type.
+
+** *Fix*: The `job.gcDelay` in the documentation is updated to have a default value of 0s.
+
+** *Result:* Users are less likely to get confused.
+
+// MGMT-20835
+* The KMM Operator Hub environment does not work because of missing MIC and MBSC CRDs.
```

Comment:
Expand MBSC on first use , use abbreviation later
```suggestion
* The KMM Operator Hub environment does not work because of missing MIC and MBSC CRDs.
```